src/components/Results/ResultTab.vue

```diff
@@ -15,10 +43,26 @@ import { detailTabs } from './use/utils'
 export default defineComponent({
   name: 'ResultTab',
   components: {
+    PageTemplate,
+    ResultHeader,
     Individual,
     Statistics,
     Spreadsheet
   },
+  props: {
+    questionnaire: {
+      type: Object as PropType<QuestionnaireByID>,
+      required: true
+    },
+    results: {
+      type: Object as PropType<ResponseResult[]>,
+      required: true
+    },
+    questions: {
+      type: Object as PropType<QuestionDetails[]>,
+      required: true
+    }
+  },
   setup() {
     const selectedTab = ref('Statistics')
```

Comment:
```suggestion
    const selectedTab = ref<DetailTabTypes>('statistics')
```
ほかのキャストしてるところも直す

これ下の`detailTabs.includes(selectedTab.value)`で赤線出てない？